docs/topics/migrations.txt

```diff
@@ -218,6 +218,28 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+----------------------
+
+.. function:: django.db.migrations.swappable_dependency(value)
+
+The ``swappable_dependency()`` function is used in migration files to specify
+a swappable dependency for a particular migration operation. It is not
+directly related to swapping models but rather influences how migrations
+handle the dependency resolution process.
+
+By using ``swappable_dependency()``, you inform the migration framework that
+the migration operation relies on a swappable model, allowing for the
+possibility of substituting the model with a different implementation in the
+future. This is typically used for referencing models that are subject to
+customization or replacement, such as the custom user model
+(``AUTH_USER_MODEL``) in Django's authentication system.
+
+In summary, ``swappable_dependency()`` is a function used in migrations to
+specify dependencies on swappable models, indicating that a migration
+operation relies on a model that can be swapped with alternative
+implementations while maintaining the same interface.
```

Comment:
```suggestion
```
Suggestion: don't summarise 2 paragraphs; it's unnecessary repetition 🙂  It kinda reminds me of when I was a kid in school trying to pad out a 1000 word essay with fillers. 😅

Reply:
ouch 😿 👍